go.mod

```diff
@@ -384,7 +385,7 @@ require (
 )
 
 // See https://github.com/moby/moby/issues/42939#issuecomment-1114255529
-replace github.com/docker/docker => github.com/docker/docker v20.10.3-0.20220224222438-c78f6963a1c0+incompatible
+replace github.com/docker/docker => github.com/docker/engine v17.12.0-ce-rc1.0.20211209213653-8955d8da8951+incompatible
```

Comment:
Looks like it downgrades the versions. It is intended?

Reply:
Its actually another package and it was auto-generated from its master branch.
Please take a look at this:
https://stackoverflow.com/questions/56032544/how-to-find-dependency-causing-sirupsen-logrus-vs-sirupsen-logrus-unexpecte

In the future we should consider upgrading to moby/moby.

Reply:
The commit you referenced is created on Dec 9, 2021. I don't think it addresses recent vulnerabilities. If scanners don't detect those vulnerabilities, it is an issue of scanners. Hiding vulnerabilities from scanners is not essential.
https://github.com/docker/engine/commit/8955d8da8951695a98eb7e15bead19d402c6eb27